circle.yml

```diff
@@ -221,42 +199,36 @@ jobs:
           name: Print working folder
           command: echo $PWD
       - run:
-          name: print global NPM cache path
-          command: echo $(npm -g bin)
+          name: print global yarn cache path
+          command: echo $(yarn global bin)
       - run:
           name: print Node version
           command: node -v
       - run:
-          name: print NPM version
-          command: npm -v
-      - run: npm run check-node-version
+          name: print yarn version
+          command: yarn -v
+      - run: yarn check-node-version
 
       ## make sure the TERM is set to 'xterm' in node (Linux only)
       ## else colors (and tests) will fail
       ## See the following information
       ##   * http://andykdocs.de/development/Docker/Fixing+the+Docker+TERM+variable+issue
       ##   * https://unix.stackexchange.com/questions/43945/whats-the-difference-between-various-term-variables
-      - run: npm run check-terminal
+      - run: yarn check-terminal
 
-      - run: npm run stop-only-all
+      - run: yarn stop-only-all
 
       - restore-caches
 
       # show what is already cached globally
-      - run: ls $(npm -g bin)
-      - run: ls $(npm -g bin)/../lib/node_modules
+      - run: ls $(yarn global bin)
+      - run: ls $(yarn global bin)/../lib/node_modules
 
-      # Install the root packages
-      # Link sup packages in ./node_modules/@packages/*
-      # Install sub packages dependencies and build all sub packages via postinstall script
       # try several times, because flaky NPM installs ...
-      - run: npm install || npm install
+      - run: yarn --ignore-engines || yarn --ignore-engines
```

Comment:
ah, now you are using ignore-engines? curious why you removed it from the .yarnrc

Reply:
It was due to the order of events during development. The good thing is that yarn will adhere to the engines, which satisfies some old scripts to check the node version, etc. The not so good thing is that the yarn version on the build agents are older than what we'd like.

I can either lower the requirements of the yarn engines value to match the lowest build agent, or possibly try to update yarn to a newer version within the docker image. The third alternative is to use `--ignore-engines`. I went with the third due to it being the path of least resistance. Ideally, I would like to **not** use `--ignore-engines` though.

Reply:
:+1: